addons/point_of_sale/models/pos_bus_mixin.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import uuid
+from odoo import fields, models
+
+class PosBusMixin(models.AbstractModel):
+    _name = "pos.bus.mixin"
+    _description = "Bus Mixin"
+
+    access_token = fields.Char('Security Token', readonly=True, default=lambda self: str(uuid.uuid4()))
+
+    def _notify(self, *notifications, private=True) -> None:
+        if isinstance(notifications[0], str):
```

Comment:
i added an example of how to call this method. This would be the type hint: `*notifications: Union[str, Tuple[str, str]]`, but this does not reflect that in the case that you want to pass just one message,  `notifications` would have to contain exactly 2 strings...  
Also, assuming that the api is clear enough, isn't it the job of the caller to check that the call is correct? 
In any case: what do you think about this? 
```py
    def _notify(self, *notifications, private=True) -> None:
        """ Send a notification to the bus.
        ex: one notification: `self._notify('STATUS', {'status': 'closed'})`
            multiple notifications: `self._notify(('STATUS', {'status': 'closed'}), ('TABLE_ORDER_COUNT', {'count': 2}))`
        """
        if isinstance(notifications[0], str):
            if len(notifications) != 2:
                raise ValueError("If you want to send a single notification, you must provide a name: str and a message: any")
            notifications = [notifications]
        self.env['bus.bus']._sendmany((self.access_token, f"{self.access_token}-{name}" if private else name, message) for name, message in notifications)
```